src/main/java/org/polystat/Analysis.java

```diff
@@ -45,4 +45,8 @@
     Iterable<String> errors(Func<String, XML> xmir,
         String locator) throws Exception;
 
+    /**
+     * @return Name of the analysis.
+     */
+    String analysisName();
```

Comment:
@Sitiritis how about just name? see this: https://www.yegor256.com/2015/01/12/compound-name-is-code-smell.html